main.go

```diff
@@ -334,7 +334,7 @@ func addCoreConfigFlagsToCommand(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&cliOptions.versionTable, "version-table", "", "", "version table name (default is public.schema_version)")
 
 	cmd.Flags().StringVarP(&cliOptions.sshHost, "ssh-host", "", "", "SSH tunnel host")
-	cmd.Flags().StringVarP(&cliOptions.sshPort, "ssh-port", "", "22", "SSH tunnel port")
+	cmd.Flags().StringVarP(&cliOptions.sshPort, "ssh-port", "", "", "SSH tunnel port")
```

Comment:
I think this is actually the only change that is needed. The problem is that CLI arguments have precedent over the config file. The default value means it always is overriding the config file.